bedrock/base/templates/includes/global-nav.html

```diff
@@ -76,31 +76,39 @@ <h3 class="summary" data-id="firefox">
                 <li>
                 {% if LANG == 'en-US' %}
                   {% set about_url = url('firefox') %}
+                  {% set desktop_text = _('Desktop') %}
+                  {% set android_text = _('Android') %}
+                  {% set ios_text = _('iOS') %}
+                  {% set focus_text = _('Focus') %}
```

Comment:
So as not to cause any confusion for the L10n team, let's not wrap these English-only strings.